Sources/StreamChatUI/ChatChannelList/ChatChannelListVC+SwiftUI.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import SwiftUI
+
+@available(iOS 13.0, *)
+/// A `UIViewControllerRepresentable` subclass which wraps `ChatChannelListVC` and shows list of channels.
+public typealias ChatChannelList = ChatChannelListVC._ChatChannelList<NoExtraData>
+
+extension _ChatChannelListVC {
+    /// A `UIViewControllerRepresentable` subclass which wraps `ChatChannelListVC` and shows list of channels.
+    public struct _ChatChannelList<ExtraData: ExtraDataTypes>: UIViewControllerRepresentable {
```

Comment:
The `<ExtraData: ExtraDataTypes>` here are defined by the enclosing `_ChatChannelListVC`. This type should be just named just `View` IMO

Reply:
Adjusted.